development/build/index.js

```diff
@@ -99,6 +99,8 @@ async function defineAndRunBuildTasks() {
       'navigator',
       'harden',
       'console',
```

Comment:
We can remove the commit that changes this file once https://github.com/MetaMask/metamask-extension/pull/20090 is merged